ts/screens/wallet/WalletHomeScreen.tsx

```diff
@@ -343,21 +347,24 @@ class WalletHomeScreen extends React.Component<Props, never> {
 const mapStateToProps = (state: GlobalState) => ({
   potWallets: walletsSelector(state),
   potTransactions: latestTransactionsSelector(state),
-  isPagoPATestEnabled: isPagoPATestEnabledSelector(state)
+  isPagoPATestEnabled: isPagoPATestEnabledSelector(state),
+  readTransactions: state.entities.transactionsRead
```

Comment:
Please create/use a selector instead of accesing the state object directly.

Reply:
updated within 7918e5770e6ccc309a80032b76d2aed136ad3c46